packages/components/form/src/form.ts

```diff
@@ -41,6 +41,8 @@ export class Form<T extends Record<string, unknown> = Record<string, unknown>> e
    */
   #value: T | undefined;
 
+  #initialValue: T | undefined;
```

Comment:
Please add a comment (even though the name is perfectly clear, we always have comments for variables)